### PR TITLE
Feature/signature

### PR DESCRIPTION
**Pull request by PeterSund**
Issue #5 
- The external signature inputfield has been removed
- Added option to enter security.txt in a textarea (option for inputfields is kept)

Issue #6 
- Input language added

Issue #7 
- Input for comments added
- Checkbox to display canonical added

General: Updated readme and i18n. Cleaned.

### CODE REVIEW

File: src/main/resources/services/securitytxt/securitytxt.js [hunk 1]
```diff
@@ -1,62 +1,97 @@
...
+    } else if (item._selected === "email" && item[item._selected].email) {
+      securityTxt += "Contact: mailto:" + item[item._selected].email + "\n";
     }
+  });
 
-    if (siteConfig.hiring) {
-        securityTxt += 'Hiring: ' + siteConfig.hiring + '\n';
-    }
+  if (siteConfig.displayCanonical) {
+    const defaultUrl = libs.portal.pageUrl({id: libs.portal.getSite()._id, type: "absolute",});
```
Comment (toredahl) [inline src/main/resources/services/securitytxt/securitytxt.js]: -skal det siste kommaet der være der? Kan godt være, så bare litt rart ut....

File: src/main/resources/site/site.xml [hunk 2]
```diff
@@ -1,109 +1,134 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <site>
     <form>
-        <option-set name="contactOptions">
-            <label i18n="contact">Contact</label>
+        <option-set name="options">
+            <label i18n="options.label">Use textarea or inputfields</label>
```
Comment (toredahl) [inline src/main/resources/site/site.xml]: pirk, men "input fields" skrives i to ord, vanligvis. Det er noen tilfeller lenger opp også.

File: src/main/resources/site/site.xml [hunk 3]
```diff
@@ -1,109 +1,134 @@
...
-                            </help-text>
+                        <input name="text" type="TextArea">
+                            <label i18n="text">Text</label>
                         </input>
                     </items>
                 </option>
-                <option name="tel">
-                    <label i18n="contact.tel.label">Phone</label>
+                <option name="fields">
+                    <label i18n="fields.label">Inputfields</label>
```
Comment (toredahl) [inline src/main/resources/site/site.xml]: her og.... Input fields

**Review by toredahl — APPROVED** [review]

**Review by matsovevada — APPROVED** [review]

File: src/main/resources/services/securitytxt/securitytxt.js [hunk 4]
```diff
@@ -1,62 +1,97 @@
...
-    });
-
-    if (siteConfig.encryption) {
-        securityTxt += 'Encryption: ' + libs.portal.pageUrl({ id: libs.portal.getSite()._id,type: 'absolute' }) + 'pgp-key.txt\n';
+// Takes the old way of configuring app, puts them in the optionset for input fields configuration and removes the old config.
+function createNewConfigOptions(siteConfig) {
+  for (let key in siteConfig) {
+    if (siteConfig.hasOwnProperty(key)) {
+      siteConfig.options.fields = siteConfig[key];
+      delete siteConfig.key;
```
Comment (Roeland-Bouvet) [inline src/main/resources/services/securitytxt/securitytxt.js]: Should this be `delete siteConfig[key];` ?

**Review by Roeland-Bouvet — APPROVED** [review]

**Pull request merged**